src/main/java/frc/robot/subsystems/arm/ArmConstants.java

```diff
@@ -1,27 +1,42 @@
 package frc.robot.subsystems.arm;
 
 import com.ctre.phoenix.motorcontrol.TalonFXInvertType;
+import edu.wpi.first.math.geometry.Translation2d;
 
-public class ArmConstants {
+public class ArmConstants { //TODO: find all constant values
 
-    // Arm constants
+    // motor configuration
     public static final double VOLT_COMP_SATURATION = 10; //[V]
     public static final boolean ENABLE_VOLT_COMPENSATION = true;
-    public static final double MOTION_ACCELERATION = 0;
-    public static final double MOTION_CRUISE_VELOCITY = 0;
-    public static final TalonFXInvertType CLOCKWISE = TalonFXInvertType.Clockwise;
+    public static final double MOTION_ACCELERATION = 0; // [rad/sec^2]
+    public static final double MOTION_CRUISE_VELOCITY = 0; // [rad/sec]
+    public static final double DEADBAND = 0.05; // [%]
+    public static final double SETPOINT_DEADBAND = 0;
+    public static final double TICKS_PER_RADIAN = 1024 / (Math.PI * 2);
+    public static final TalonFXInvertType MAIN_CLOCKWISE = TalonFXInvertType.Clockwise;
+    public static final TalonFXInvertType AUX_CLOCKWISE = TalonFXInvertType.Clockwise;
```

Comment:
It's better for these names to include "invert type", not the specific invert type used. Because if you'll use another invert type in the future, the names of the constants will be irrelevant.
Also, perhaps using `TalonFXInvertType.FollowMaster` is better for the auxiliary motor.